### DISCUSSION TITLE
Doing tests on a 0V Cell and Connected in series with other cells not equal to 0V

### DISCUSSION THREAD

**Discussion by Azizbs1130** [pybamm-team/PyBaMM#3014] [Q&A]
Hello Everyone,

I am working on a subject in which lets say we have 6 series of battery cells conected and i want to short cut only one cell (discharge it to 0V) in reality it will be like installing a bridge on it to bypass it in case its damaged so the whole pack of cell keep functioning.

So the questions are:
1) is it possible to do some work on a 0V cell in pybamm, because from what i noticed it crashes when we reach minimum voltage?
2) is it possible to do a simulation of a series of batteries or parallel? and get for example voltage and other informations of each cell seperatly?

**Comment by brosaplanella** [comment]
Discharging beyond the minimum voltage is likely to crash the solver. You can try it via an experiment, but not sure this is the best way of modelling a shortcircuit. I don't think we have any way of simulating internal shortcircuits in PyBaMM. This would require adding some extra equations for the separator, so we can treat it as a conducting material.
